### ISSUE TITLE
Add  Shortcode as a method of fetching data like @username & #hashtag

### ISSUE THREAD

**Issue by AriaShishegaran** [instaloader/instaloader#129]
Is there a way to get a certain post's metadata just like what we do with hashtags and usernames, since it is also a unique parameter and it would make it much easier (in contrast to using the python modules to program it to receive a certain short code's metadata)

What i have in mind would be sth like this:
`instaloder --comments "Bi9jw-dH2j-"`

what do u think?

**Comment by nagualcode** [comment]
I second this.
Sometimes I only need profile_pic_url and username

**Comment by aandergr** [comment]
How could the shortcode target be encoded such that Instaloader would not confuse it with a profile's name?

**Comment by AriaShishegaran** [comment]
Maybe with a parameter like instaloader -X POST URL so that i lt knows it
is for fetxhing a post not a page

**Comment by aandergr** [comment]
Implemented in `v4.1-dev`.

```
pip install --upgrade git+https://github.com/instaloader/instaloader@v4.1-dev
```
